charts/stable/hastebin/chart/README.md

```diff
@@ -0,0 +1,116 @@
+# ${CHARTNAME}
```

Comment:
Run helm docs

Reply:
some weirdness happened, there should not be two hastebins charts

Reply:
removed duplicate